easy-random-core/src/main/java/org/jeasy/random/RandomizationContext.java

```diff
@@ -148,4 +151,12 @@ public Object getRootObject() {
     public EasyRandomParameters getParameters() {
         return parameters;
     }
+
+    @Override
+    public <T> T randomElementOf(List<T> list) {
```

Comment:
This should be removed as explained below.

Reply:
removed